commands/partner_interconnect_attachment.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// Network creates the network command.
+func Network() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "network",
+			Short:   "Display commands that manage network products",
+			Long:    `The commands under ` + "`" + `doctl network` + "`" + ` are for managing network products`,
+			GroupID: manageResourcesGroup,
```

Comment:
```suggestion
			GroupID: manageResourcesGroup,
			Hidden:  true,
```

What's the release status on this? If it is in private preview and only available to a small set of customers, we should hide it from top-level help output to prevent confusion.

Reply:
Yes, still available to a small set of customers.
Addressed